### PR TITLE
improve load plugin

### PR DESCRIPTION
**Pull request by qiweiii**
related: #512

- only fix the rpc plugin load part
- for cli plugin, seems need to add another plugin command, so we can load cli plugin on-demand as sub-commands, but this will bring breaking change to how user use plugin cli

### CODE REVIEW

**Comment by ermalkaleci** [comment]
if we do this then we need to a way to know the commands `-h` should tell you the commands also `rpc_methods` should return the methods

File: packages/chopsticks/src/plugins/index.ts [hunk 1]
```diff
@@ -1,45 +1,51 @@
...
+export const rpcPluginHandlers: Handlers = {}
 
 // list of plugins directory
 const plugins = readdirSync(new URL('.', import.meta.url)).filter((file) =>
   lstatSync(new URL(file, import.meta.url)).isDirectory(),
 )
 
+// find all rpc exports
+export const rpcPluginMethods = readdirSync(new URL('.', import.meta.url))
+  .filter((file) => existsSync(new URL(`${file}/rpc.ts`, import.meta.url)))
```
Comment (ermalkaleci) [inline packages/chopsticks/src/plugins/index.ts:19]: this won't work when it's build

Comment (ermalkaleci) [reply packages/chopsticks/src/plugins/index.ts:19]: don't use exists. use lstatSync instead and see if there's a filename with rpc

File: executor/package.json [hunk 2]
```diff
@@ -8,8 +8,8 @@
 		"url": "https://github.com/AcalaNetwork/chopsticks"
 	},
 	"scripts": {
-		"clean": "rm -rf web node",
-		"build": "wasm-pack build --target web --out-dir browser; wasm-pack build --target nodejs --out-dir node; scripts/pack-wasm.cjs"
+		"clean": "rm -rf browser node",
+		"build": "npx wasm-pack build --target web --out-dir browser && npx wasm-pack build --target nodejs --out-dir node && scripts/pack-wasm.cjs"
```
[the remark this answers is not in the document]
Comment (qiweiii) [reply executor/package.json:12]: When work in dev container, it says wasm-pack not found

File: packages/chopsticks/src/plugins/index.ts (hunk 1 above)
Comment (qiweiii) [reply packages/chopsticks/src/plugins/index.ts]: both `exist` and `lstatSync` won't work, they both require full filename with extension, so I changed to filter by file name instead

File: executor/package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (ermalkaleci) [reply executor/package.json:12]: yarn install should install it

**Pull request merged**